backend/CuratedLists/views.py

```diff
@@ -137,11 +137,12 @@ def add_curation(request):
         try:
             if validate_session(request):
                 curation = json.loads(request.body.decode('utf-8'))
-                # perform dbaction()
+                cur = Curation.objects.create(author=Author.objects.get(username=curation['username']), topic=curation['topic'],
+                data=curation['data'])
```

Comment:
Not required, we should just create it in the db and return a `HTTPStatus.CREATED`.